R/class.R

```diff
@@ -0,0 +1,43 @@
+inject_super <- function(fun) {
+  e <- new.env()
+  e$super <- function() {
+    bt <- reticulate::import_builtins()
+    self <- as.list(parent.frame())$self
```

Comment:
Prefer instead using:

```
get("self", envir = parent.frame(), inherits = FALSE)
```

As this would avoid attempting to evaluate other symbols defined in the parent frame.